enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1574,6 +1579,7 @@ func newPartitionEvictor(part disk.Partition, fileStorer filestore.Store, blobDi
 	pe.sizeBytes = sizeBytes
 	pe.casCount = casCount
 	pe.acCount = acCount
+	pe.lru.UpdateSizeBytes(sizeBytes)
```

Comment:
this is a tiny fix -- previously we were not updating the lru size until a write happened -- now we also update it when reading the cached size from the db (or if it was force recomputed).